core/src/validator.rs

```diff
@@ -527,6 +527,7 @@ impl Validator {
         start_progress: Arc<RwLock<ValidatorStartProgress>>,
         socket_addr_space: SocketAddrSpace,
         use_quic: bool,
+        vote_use_quic: bool,
         tpu_connection_pool_size: usize,
         tpu_enable_udp: bool,
```

Comment:
all these same type `: bool` arguments next to each other are very bug prone and we actually had bugs where the arguments were ordered erroneously.
can we use a struct or something here to avoid that?

Reply:
I encapsulated this into a struct.